### ISSUE TITLE
Send alerts to the terminal on events

### ISSUE THREAD

**Issue by mathstuf** [nbedos/citop#9]
Probably should be a configuration setting, but making a bell on the terminal when an event happens would be great (I'm using GitLab terminology here, I'm not too worried about naming):

  - pipeline-started
  - pipeline-failed
  - pipeline-completed
  - job-started
  - job-failed
  - job-completed

**Comment by nbedos** [comment]
I agree that this would be useful but it is currently not possible without fixing https://github.com/gdamore/tcell/issues/2 first. [This PR](https://github.com/gdamore/tcell/pull/293) should be enough for Unix systems so I'll probably have to make a private fork of `tcell` that includes this change.

I'll wait a bit before working on this though, in case `tcell` has other limitations that would make using `ncurses` a better choice.
